src/pages/Login/Login.scss

```
@@ -0,0 +1,164 @@
+    margin-top: 200px;
+    font-size: 25px;
+    color: rgb(244, 187, 95);
+    font-weight: bold;
+  }
+  .inputWrap {
+    display: flex;
+    align-items: center;
+    flex-direction: column;
+    margin-top: 90px;
+
+    .loginWrapper {
+      position: relative;
+      .inputId,
+      .inputPw {
+        font-size: 15px;
+        width: 250px;
+        height: 30px;
+        margin-bottom: 50px;
+        text-align: center;
+        border: none;
+        border-bottom: 1px solid;
+
+        &:focus + .separator {
+          transform: scaleX(1) translateY(-2px);
+          opacity: 1;
+        }
+
+        &:hover {
+          border-bottom: 2px solid rgb(95, 95, 95);
+        }
+        &:focus {
+          outline: none;
+        }
+      }
+
+      .separator {
+        height: 2px;
+        width: 100%;
+        background: rgb(67, 80, 175);
+        display: block;
+        transform: scaleX(0) translateY(-2px);
+        transform-origin: 50%;
+        transition: all 0.15s linear;
+        position: absolute;
+        top: 31px;
+      }
+    }
+
+    .topEmail,
+    .topPassword {
+      font-size: 20px;
+      font-weight: bold;
+      color: rgb(112, 112, 112);
+      margin-bottom: 30px;
+    }
+    .loginButton {
+      width: 150px;
+      height: 45px;
+      border-radius: 10px;
+      color: rgb(178, 178, 178);
+      border: 1px solid rgb(178, 178, 178);
+      font-weight: bold;
+      font-size: 18px;
+      background-color: transparent;
+    }
+    .loginButtonActive {
+      width: 150px;
+      height: 45px;
+      border-radius: 10px;
+      color: white;
+      border: 1px solid rgb(178, 178, 178);
+      font-weight: bold;
+      font-size: 18px;
+      background-color: #f4bb5f;
+      cursor: pointer;
+    }
+  }
+  .socialLogin {
+    display: flex;
+    align-items: center;
+    flex-direction: column;
+    margin-top: 50px;
+
+    .kakaoLoginButton {
+      width: 190px;
+      height: 50px;
+      display: flex;
+      justify-content: space-evenly;
+      align-items: center;
+      font-size: 15px;
+      margin-bottom: 20px;
+      border-radius: 10px;
+      background-color: rgb(255, 232, 18);
+      border: none;
+
+      .fa-comment {
+        color: #59500e;
+      }
+    }
+    .naverLoginButton {
+      width: 190px;
+      height: 50px;
+      display: flex;
+      justify-content: space-evenly;
+      align-items: center;
+      font-size: 15px;
+      color: white;
+      margin-bottom: 20px;
+      border-radius: 10px;
+      background-color: #37c736;
+      border: none;
+
+      .fa-neos {
+        color: white;
+      }
+    }
+    .googleLoginButton {
+      width: 190px;
+      height: 50px;
+      display: flex;
+      justify-content: space-evenly;
+      align-items: center;
+      font-size: 15px;
+      color: rgba(0, 0, 0, 0.54);
+      margin-bottom: 20px;
+      border-radius: 10px;
+      border: 0.5px solid rgb(178, 178, 178);
+      background-color: transparent;
+    }
```

button에 관련되는 동일한 속성을 가지는 className을 따로 빼고, 각각 달라지는 부분만 따로 className 생성하는 방식으로 관리하면 훨씬 깔끔할 것 같습니다.
```suggestion
    .socialLoginButton {
      ...
      
      .kakao {
        ...
      }
      
      .naver {
        ...
      }
      
      .google {
        ...
      } 
    }
```

---

css 지옥에서 꺼내주세요 ㅜㅜ